server/src/domain/sql_group_backend_handler.rs

```diff
@@ -114,6 +117,7 @@ impl GroupListerBackendHandler for SqlBackendHandler {
                 }
             })
             .collect();
+        let schema = self.get_schema().await?;
```

Comment:
Can you add a TODO to wrap all that in a transaction? Same with users.

Reply:
Done